var/spack/repos/builtin/packages/libmesh/package.py

```diff
@@ -15,6 +15,10 @@ class Libmesh(AutotoolsPackage):
     url      = "https://github.com/libMesh/libmesh/releases/download/v1.0.0/libmesh-1.0.0.tar.bz2"
     git      = "https://github.com/libMesh/libmesh.git"
 
+    version('develop', branch='master', submodules=True)
```

Comment:
Now that #1983 has been merged, you can call this version `master` if you want.

Reply:
Thanks for the review @adamjstewart . 
I have made the modification and resubmitted.